packages/design-picker/src/components/style-variation-badges/utils.ts

```diff
@@ -112,8 +115,7 @@ export function getStylesColorFromVariation(
 	try {
 		return { background: colorBase, text: findColorBestAnalogous( colorList, colorBase ) };
 	} catch ( e ) {
-		// eslint-disable-next-line no-console
-		console.error( e, variation );
+		debug( e instanceof Error ? e.message : e, variation );
```

Comment:
I think it's good to always use `debug( e, variation )`; it would give us nice stacktrace, what do you think?

<img width="1016" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/54c59801-7c96-432d-abe7-b2387e5ddb39">

Reply:
Sure, why not 😀